app/services/ocrService.py

```
@@ -19,25 +21,104 @@
 
 # 바이트 + MIME 타입을 받아 gemini로 OCR을 수행 -> 텍스트를 json 형태로 반환
 def ocr_bytes_to_pdrecord_json(
     file_bytes: bytes,
     content_type: str,
     model: str = "gemini-2.5-flash",
+    save_to_gcs: bool = False,
+    user_hash: Optional[str] = None,
+    gcs_filename: Optional[str] = None,
 ) -> Dict[str, Any]:
     if not file_bytes:
         raise OcrError("빈 파일입니다.")
```

Ruff (0.14.4)

Avoid specifying long messages outside the exception class

(TRY003)